handlers/_swagger.js

```diff
@@ -19,7 +19,8 @@ var polymorphicValidation;
 
 exports.init = function (app, auth, config, logger, serviceLoader, swagger, callback) {
     var cfg = config.get('swagger');
-    responseModelValidationLevel = /error|warn/.test(cfg.validateResponseModels) ? cfg.validateResponseModels : 0;
+    
+    responseModelValidationLevel = /error|warn|fail/.test(cfg.validateResponseModels) ? cfg.validateResponseModels : 0;
```

Comment:
calculated swagger config could be done once in the swagger service and then exported for use here (and potentially elsewhere)
... see the comment in the swagger service